site/docs/reference/Connectors/capture-connectors/apache-kafka.md

```diff
@@ -0,0 +1,76 @@
+All SASL methods are permissable for clusters used with Flow, so long as connection details are provided.
+When authentication details are not provided, the client connection will attempt to use PLAINTEXT (insecure) protocol.
+
+If you don't already have authentication enabled on your cluster, Estuary recommends [SASL/SCRAM](https://kafka.apache.org/documentation/#security_sasl_scram).
+With SCRAM, you set up a username and password, making it analogous to the traditional authentication mechanisms
+you use in other applications.
+
+For connection security, Estuary recommends that you enable TLS encryption for your SASL mechanism of choice,
+as well as all other components of your cluster.
+Note that because TLS replaced now-deprecated SSL encryption, Kafka still uses the acronym "SSL" to refer to TLS encryption.
+See [Confluent's documentation](https://docs.confluent.io/platform/current/kafka/authentication_ssl.html) for details.
+
+:::info Beta
+TLS encryption is currently the only supported connection security mechanism for this connector.
+Other connection security methods may be enabled in the future.
+:::
+
+## Configuration
+
+There are various ways to configure and implement connectors. See [connectors](../../../concepts/connectors.md#using-connectors) to learn more about these methods. The values and code sample below provide configuration details specific to the Apache Kafka source connector.
+
+### Values
+
+| Value | Name | Description | Type | Required/Default |
+|---|---|---|---|---|
+| `bootstrap_servers` | Bootstrap servers | The initial servers in the Kafka cluster to connect to. The Kafka client will be informed of the rest of the cluster nodes by connecting to one of these nodes. | array | Required |
+| `tls`| TLS | TLS connection settings | string | `"system_certificates"` |
+| `authentication`| Authentication | Connection details used to authenticate a client connection to Kafka via SASL | null, object | |
+| `authentication/mechanism` | Mechanism | SASL mechanism describing how to exchange and authenticate client servers | string |  |
+| `authentication/password` | Password | Password, if applicable for the authentication mechanism chosen | string | |
+| `authentication/username` | Username | Username, if applicable for the authentication mechanism chosen | string | |
+
+### Sample
+```yaml
+captures:
+  ${TENANT}/${CAPTURE_NAME}:
+    endpoint:
+      connector:
+        image: ghcr.io/estuary/source-kafka:dev
+        config:
+            bootstrap_servers: localhost:9093
+            tls: system_certificates
+            authentication: #FOR REVIEW: need semi-realistic example values
+                mechanism:
+                username:
+                password:
+    bindings:
+      - resource:
+           namespace: ${STREAM_NAMESPACE} #FOR REVIEW: does namespace matter?/What does it do here?
```

Comment:
I think `namespace` is an optional Airbyte-spec thing. I don't believe we use it for anything.